lib/test/unit_tests/test_adf_base.py

```diff
@@ -131,17 +141,22 @@ def test_AdfBase_debug_write(self):
         #Create AdfBase object with debug setting:
         adf_test = AdfBase(debug=True)
 
+        #Grab debug log name
+        debug_fname = adf_test.debug_fname
+
         #Call "debug_log" method:
         adf_test.debug_log("test")
 
+        print(debug_fname)
+
```

Comment:
I would remove this print statement here (as I think we were only using it temporarily for debugging purposes).